src/eval.c

```diff
@@ -234,7 +234,7 @@ void scriptingInit(int setup) {
     {
         char *errh_func =       "local dbg = debug\n"
                                 "debug = nil\n"
-                                "function __redis__err__handler(err)\n"
+                                "function __server__err__handler(err)\n"
```

Comment:
I would suggest aliasing __server__err__handler to __redis__err__handler to address Madelyn's compat concern.

Reply:
fixed in next commit.